### DISCUSSION TITLE
Quill Editor can't do br line breaks

### DISCUSSION THREAD

**Discussion by pdclark** [pods-framework/pods#6812] [Ideas]
## Problem

As reported here: https://wordpress.org/support/topic/quill-editor-does-not-support-hyperlinks-or-line-breaks/

- [x] Quill editor does not have button for inserting links enabled — fix for this in PR #6221 .
- [ ] Quill editor converts line-breaks (`<br/>` / `shift-return`) to paragraphs (`<p>` / `return`). Possibly related: https://podswp.slack.com/archives/C02SVLHQF/p1634763507119200 , but adding `remove_filter( 'pods_template_content', 'pods_clean_linebreaks' );` doesn't seem to resolve the problem.
- [ ] #6224

## Proposed Solution

- For links, Quill takes an argument that can be added.
- For line breaks, the code `remove_filter( 'pods_template_content', 'pods_clean_linebreaks' );` doesn't resolve the problem.
- May need help for someone like @zrothauser to configure an HTML button, if that's desired... see https://github.com/benwinding/quill-html-edit-button

## Screenshots / Screencast

<img width="1045" alt="Screen Shot 2021-10-22 at 10 19 59 AM" src="https://user-images.githubusercontent.com/126889/138497119-3a323b20-a6d1-42eb-8f46-ac8044edc391.png">

**Comment by sc0ttkclark** [comment]
For `<br>` tags, this is an interesting solution: https://github.com/zenoamaro/react-quill/issues/513#issuecomment-523783053

Related: https://github.com/quilljs/quill/issues/1074 and https://github.com/quilljs/quill/issues/3161

**Comment by sc0ttkclark** [comment]
It's possible that our best bet may be to advise people that the Quill editor itself only supports `<p>` tags, it does not support line breaks `<br>` and that's that.

**Comment by sc0ttkclark** [comment]
For some reason the Quill people are anti-`<br>` tag and they expect `<p>` tags to have no margins on them. But not in the WP world.

I'm open to setting up an alternative WYSIWYG editor option too, a bit bummed that Quill failed hard in this spot.
